src/TBufferStyle.cpp

```diff
@@ -0,0 +1,1262 @@
+                //    break;
+                // case 64: // ideogram stress marking
+                //    break;
+                // case 65: // cancels the effects of 60 to 64
+                //    break;
+                case 90:
+                    mFgColor = mLightBlack;
+                    mFgColorLight = mLightBlack;
+                    mIsDefaultColor = false;
+                    break;
+                case 91:
+                    mFgColor = mLightRed;
+                    mFgColorLight = mLightRed;
+                    mIsDefaultColor = false;
+                    break;
+                case 92:
+                    mFgColor = mLightGreen;
+                    mFgColorLight = mLightGreen;
+                    mIsDefaultColor = false;
+                    break;
+                case 93:
+                    mFgColor = mLightYellow;
+                    mFgColorLight = mLightYellow;
+                    mIsDefaultColor = false;
+                    break;
+                case 94:
+                    mFgColor = mLightBlue;
+                    mFgColorLight = mLightBlue;
+                    mIsDefaultColor = false;
+                    break;
+                case 95:
+                    mFgColor = mLightMagenta;
+                    mFgColorLight = mLightMagenta;
+                    mIsDefaultColor = false;
+                    break;
+                case 96:
+                    mFgColor = mLightCyan;
+                    mFgColorLight = mLightCyan;
+                    mIsDefaultColor = false;
+                    break;
+                case 97:
+                    mFgColor = mLightWhite;
+                    mFgColorLight = mLightWhite;
+                    mIsDefaultColor = false;
+                    break;
+                case 100:
+                    mBgColor = mLightBlack;
+                    break;
+                case 101:
+                    mBgColor = mLightRed;
+                    break;
+                case 102:
+                    mBgColor = mLightGreen;
+                    break;
+                case 103:
+                    mBgColor = mLightYellow;
+                    break;
+                case 104:
+                    mBgColor = mLightBlue;
+                    break;
+                case 105:
+                    mBgColor = mLightMagenta;
+                    break;
+                case 106:
+                    mBgColor = mLightCyan;
+                    break;
+                case 107:
+                    mBgColor = mLightWhite;
+                    break;
+                default:
+                    qDebug().noquote().nospace() << "TBuffer::translateToPlainText(...) INFO - Unhandled single SGR code sequence CSI " << tag << " m received, Mudlet will ignore it.";
+                }
+            }
+        }
+    }
+}
+
+
+void TBufferStyle::decodeSGR38(const QStringList& parameters, bool isColonSeparated)
+{
+#if defined(DEBUG_SGR_PROCESSING)
```

Comment:
:warning: You will have to move the place where this preprocessor symbol is (potentially) declared from the top of the `TBuffer.cpp` file to somewhere else - either a header (`.h`) file which all the (`.cpp`) that test it `#include` it - or as something in the project (the correct `CMakeList.txt` - not sure myself which one for CMake and the `mudlet.pro` for  QMake) files.

Reply:
Makes sense. Is it this macro defined below in this snippet? Was it supposed to be commented?
https://github.com/Mudlet/Mudlet/blob/3923dec40f06fdf3a17e8cf7a3dbb073edd39f00/src/TBuffer.cpp#L45-L46

Reply:
Yeah, so we'd uncomment when debugging and otherwise not get the spam.